pkgs/top-level/all-packages.nix

```diff
@@ -26143,7 +26143,11 @@ with pkgs;
   linkerd_edge = callPackage ../applications/networking/cluster/linkerd/edge.nix { };
   linkerd_stable = linkerd;
 
-  kumactl = callPackage ../applications/networking/cluster/kumactl { };
+  kuma = callPackage ../applications/networking/cluster/kuma { };
+  kumactl = callPackage ../applications/networking/cluster/kuma {
+    isFull = false; pname = "kumactl"; };
```

Comment:
Please put the closing } on the next line

Reply:
Thank's for review. I wish it is good now